inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/config/model/metrics/ClassLoaderMetricsSettings.java

```diff
@@ -0,0 +1,28 @@
+package rocks.inspectit.oce.core.config.model.metrics;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.NonNull;
+
+import java.time.Duration;
+import java.util.Map;
+
+/**
+ * Settings for the @{@link rocks.inspectit.oce.core.metrics.ClassLoaderMetricsRecorder}.
+ */
+@Data
+@NoArgsConstructor
+public class ClassLoaderMetricsSettings {
+
+    /**
+     * Contains the enabling flag for each metric.
+     */
+    Map<String, Boolean> enabled;
```

Comment:
I notice you omitted `private` in all config model classes, any specific reason?